google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/features/InstanceApi.java

```diff
@@ -237,6 +237,7 @@ Operation setDiskAutoDelete(@PathParam("instance") String instanceName,
     * Sets an instance's scheduling options.
     * @see <a href = "https://cloud.google.com/compute/docs/instances#onhostmaintenance"/>
     *
+    * @deprecated See {@link #setScheduling(String, org.jclouds.googlecomputeengine.domain.Instance.Scheduling.OnHostMaintenance, boolean, boolean)}. Will be removed in 2.0
```

Comment:
@nacx: Now with deprecation warning on the old version. Leave the `@see` on the new version? Or would it be better to avoid references back to deprecated calls?